### ISSUE TITLE
SynthDef specs not returning expected values

### ISSUE THREAD

**Issue by dmorgan-github** [supercollider/supercollider#4816]
## Environment

* SuperCollider version: 3.11.0-beta1 (latest build)
* Operating system: Mac 10.14.5
* Other details (Qt version, audio driver, etc.): N/A

## Steps to reproduce
Specs configured in a synthdef's metadata section don't appear to return the configured values as expected.

```supercollider
// 3.11.0-beta1 (latest build)
(
SynthDef(\test, {
	var val = \val.kr(1);
	var sig = SinOsc.ar;
	Out.ar(\out.kr(0), sig);
}, metadata: (
	specs: (
		val: [0, 3, \lin, 0, 1, "val"].asSpec,
	)
)
).add
)
SynthDescLib.global.at(\test).metadata.specs[\val];
// -> a ControlSpec(0.0, 1.0, 'linear', 0.0, 0.0, "")
```
in 3.10.2
```supercollider
// v.3.10.2
SynthDescLib.global.at(\test).metadata.specs[\val];
// -> a ControlSpec(0, 3, 'linear', 0, 1, "val")
```
## Expected vs. actual behavior
Behavior is correct in 3.10.2 but not correct in 3.11.0-beta1 (latest build)

**Comment by jamshark70** [comment]
https://github.com/supercollider/supercollider/pull/3814 seems, accidentally, to require NamedControl specs to be passed into `\name.kr(...)`.

https://github.com/supercollider/supercollider/blob/c4824dbe62f6558908feb8d94720f4b7627deb15/SCClassLibrary/Common/Control/GraphBuilder.sc#L81

Here, the logic is "use the spec passed in." It should have a branch for "if the passed-in spec is nil, check metadata specs."

Let's fix this for the release. Should be easy and this is a quite bad breakage.

**Comment by brianlheim** [comment]
closed in #4817, thanks @jamshark70 !
